code/game/machinery/ARES/ARES_interface_apollo.dm

```diff
@@ -387,28 +411,45 @@
 			access_ticket.ticket_status = TICKET_REJECTED
 			to_chat(usr, SPAN_NOTICE("[access_ticket.ticket_type] [access_ticket.ticket_id] marked as rejected."))
 			ares_apollo_talk("Access Ticket [access_ticket.ticket_id]: [access_ticket.ticket_submitter] was rejected access by [last_login].")
+			for(var/obj/item/card/id/identification in link.waiting_ids)
+				if(!istype(identification))
+					continue
```

Comment:
this istype is redundant, as you are iterating through the list without `as anything`, so it will *all* be of type /obj/item/card/id